examples/cross_decomposition/plot_pcr_vs_pls.py

```diff
@@ -0,0 +1,116 @@
+"""
+==================================================================
+Principal Component Regression vs Partial Least Squares Regression
+==================================================================
+
+This example compares `Principal Component Regression
+<https://en.wikipedia.org/wiki/Principal_component_regression>`_ (PCR) and
+`Partial Least Squares Regression
+<https://en.wikipedia.org/wiki/Partial_least_squares_regression>`_ (PLS) on a
+toy dataset. Our goal is to illustrate how PLS can outperform PCR when the
+target is strongly correlated with some features that have a low variance.
+
+PCR consists in applying :class:`~sklearn.decomposition.PCA` to the
```

Comment:
```suggestion
PCR consists of applying :class:`~sklearn.decomposition.PCA` to the
```

Reply:
I think consists in is correct: https://github.com/scikit-learn/scikit-learn/pull/17151#discussion_r421565456

Reply:
Both seems valid.

From the link, "consist of" means

> to be composed or made up of

In our case: "PCR is composed of applying PCA"

From the link. "consists in" means:

> To have the thing mentioned as the only or most important part.

In our case: "PCA is an importart part of PCR."

Reply:
Ok, thanks. I updated the sentence